### DISCUSSION TITLE
Is ASM available?

### DISCUSSION THREAD

**Discussion by doubletaro** [jvde-github/AIS-catcher#111] [General]
Hello, everyone.

I want to get and also analize ASM data.
Is ASM available?

Thanks in advance.

**Comment by jvde-github** [comment]
Hello!

Are you referring to AIS messages 6 and 8? The program can receive these if you are in a location where they are broadcasted and produce the necessary NMEA. However, the ability to decode this to JSON and use this in the webclient is limited.

Does this answer your question?

**Comment by doubletaro** [comment]
Thank you for your reply!
I understand that your code can receive ASM messages from ships in NMEA format.
Can I take this to the beach and read her ASM messages sent from nearby ships?

**Comment by jvde-github** [comment]
You can use the software with some hardware to receive AIS messages from ships in NMEA format, for example with a  portable solution near a beach. I am not sure though how often ships broadcast AIS ASM messages (I dont receive them often) but that is something to try.

**Comment by doubletaro** [comment]
Sorry for the late reply. Because I was on a business trip.

First of all, I would like to test whether it can be received on android!!
Can you make software for the sending side?

**Reply by jvde-github** [reply]
Hello, Android version can also receive AIS. You need to combine with an app like boatbeacon or opencpn to visualize the result.

No plans for making an app that can send ais messages.

**Reply by doubletaro** [reply]
Thanks for your advice,
Anyway, I'll try it!
